### ISSUE TITLE
Archived apps on Android 15

### ISSUE THREAD

**Issue by bagarwa** [d4rken-org/sdmaid-se#1191]
I'm on Pixel 7 Pro running Android 15 Beta 2.

SDMaid SE 1.0.0-rc0.

I ran corpse finder and it picked up a bunch of remains whose package names look like apps that I archived. I think if I delete these, and later restore the archived apps, they won't have their data. (Unless archived apps save their data to cloud too.)

Is my assumption correct? If so, how do I exclude archived apps' data from corpse finder and others? Just excluding /data_mirror/ directory is enough or something else is needed?

I see same package names are also showing up under /Android/data too. Surely that can't be excluded because normal apps also host their data there. What do I do?

**Comment by d4rken** [comment]
By "archived" you mean the new Android 15 feature right?

I have not looked into this yet, but excluding found results by pressing "exclude" on the items should just work.

The more interesting question is this:

> I think if I delete these, and later restore the archived apps, they won't have their data. (Unless archived apps save their data to cloud too.)

I'll try to find some time to test this, or can you test it too?

Are you rooted or unrooted?

If the data can't be restored after deleting the corpses (and is necessary), then we need to find a way to detect this case and either exclude them or at least hide them behind a setting :thinking:

**Comment by bagarwa** [comment]
Yep, this is about the app archiving feature added in Android 15. I am rooted with magisk. 

I now tested this for Zoom. I was logged in in the app. Archived it. Restored it. I was still logged in. 
I archived it again. Then ran Corpse Finder. It found zoom app's data in 3 places - 

- /data_mirror/data_ce/null/0/us.zoom.videomeetings
- /data_mirror/data_de/null/0/us.zoom.videomeetings
- /storage/emulated/0/Android/data/us.zoom.videomeetings

I deleted the first two. Then restored Zoom. This time it launched as a fresh install, and I wasn't logged in. i.e. no data restored.
